src/commands/status/host.js

```diff
@@ -11,7 +11,9 @@ class HostStatusCommand extends BaseCommand {
    * @return {Promise<void>}
    */
   async runWithDependencies() {
-    const rows = [];
+    const rows = [
```

Comment:
Do you think we need such headers? :)) I feel like it's pretty obvious so we can just skip them :)

Reply:
Actually I didn't think so, but we are adding a header row to the table so it made sense to put actual headers in it ;) I have removed the header row completely now, is this better?